From f6b5f75856f85c2232bff6eee2fdcaf05907bbe3 Mon Sep 17 00:00:00 2001
From: adorfer <adorferen@gmail.com>
Date: Tue, 13 Oct 2026 18:43:47 +0200
Subject: [PATCH 14/42] uboot-envtools: qualcommax: add ipq50xx skeleton

Add the qualcommax_ipq50xx defaults script without device entries, in the
layout used by OpenWrt main on Linux 6.6 (8dbbca029d^, there under
package/boot/uboot-tools/uboot-envtools/files/). Devices add their
U-Boot environment location on top.
---
 .../boot/uboot-envtools/files/qualcommax_ipq50xx | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 package/boot/uboot-envtools/files/qualcommax_ipq50xx

diff --git a/package/boot/uboot-envtools/files/qualcommax_ipq50xx b/package/boot/uboot-envtools/files/qualcommax_ipq50xx
new file mode 100644
index 0000000000..19f0f415c7
--- /dev/null
+++ b/package/boot/uboot-envtools/files/qualcommax_ipq50xx
@@ -0,0 +1,16 @@
+[ -e /etc/config/ubootenv ] && exit 0
+
+touch /etc/config/ubootenv
+
+. /lib/uboot-envtools.sh
+. /lib/functions.sh
+
+board=$(board_name)
+
+case "$board" in
+esac
+
+config_load ubootenv
+config_foreach ubootenv_add_app_config
+
+exit 0
-- 
2.53.0

